docs/series/LAMP-server.md

```diff
@@ -0,0 +1,241 @@
+
+* **Make a user in which the site would belong to and add the needed directories:**
+    ```sh
+    useradd almasite
+    cd /home/almasite/
+    mkdir logs public_html
+    ```
+
+* **Fix the privileges:**
+    ```sh
+    chown almasite:almasite logs public_html
+    ```
+
+* **Create a configuration file in `/etc/httpd/conf.d/` named almasite.conf:**
+    ```sh
+    <VirtualHost *:80>
+        # The domain
+        ServerName almasite.com
+        # The domains alias
+        ServerAlias www.almasite.com
+        # Full path to the website files
+        DocumentRoot /home/almasite/public_html
+        # Full path to the error log file
+        ErrorLog /home/almasite/logs/error.log
+        # A custom log to record requests to the website
+        CustomLog /home/almasite/logs/access.log combined
+        # Apply directives only on a given directory
+        <Directory /home/almasite/public_html>
+            # Enables the server to follow symbolic links within directories
+            Options Indexes FollowSymLinks
+            # Permits the use of .htaccess files to override server configuration settings
+            AllowOverride All
+            # Allow access to the specified directory and its contents to all users
+            Require all granted
+            # Specifies default file for directory access
+            DirectoryIndex index.php
+        </Directory>
+        # Route requests based on patterns
+        ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://127.0.0.1:9000/home/almasite/public_html/
+    </VirtualHost>
+    ```
+    Notice the comments explaining each directive.
+
+* **Add a php-fpm pool**
+    ```sh
+    cp /etc/php-fpm.d/www.conf /etc/php-fpm.d/almasite.conf
+    ```
+
+    Open the newly created file with your preferred text editor and change the following:
+    - **\[www\]** - with the pools name (.e.g almasite)
+    - **user = apache** - your user (.i.e almasite)
+    - **group = apache** - your group (.i.e almasite)
+    - **listen = /run/php-fpm/www.soc** - your listening method, in this case we use port 9000 (listen = 9000)
+
+* **Add the websites files: (or file in this case ;))**
+    ```sh
+    touch /home/almasite/public_html/index.php
+    ```
+
+    Open your preferred text editor and add the following:
+    ```html
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>Hello world!</title>
+        </head>
+        <body>
+            <?php $now = date("H:i:s", time()); echo "<h1>Hello world!<br>This message is sent from PHP, the time now is $now</h1>" ?>
+        </body>
+    </html>
+    ```
+
+* **Manipulate your hosts file to view your newly created website:**
+
+    ::: tip
+    While I cant tell you your **IP** address, I can tell you that the domain is almasite.com and the alias is www.almasite.com. (That is if you followed my guide exactly)
+
+    ```sh
+    YOUR-IP-ADDRESS www.almasite.com almasite.com
+    ```
+    :::
+
+    <br>
+
+    *<ins>Linux</ins>:*
```

Comment:
These HTML tags seem out of place?

Reply:
I just wanted to underline the text, I wasn't sure how to do so and I figured I just do it like that :smile: If you have a better way by all means change it (or tell me how).
The `<br>` was just for spacing, we can remove it if it's an issue.

Reply:
Generally you'd want to avoid using HTML in markdown documents unless absolutely necessary. I didn't realize markdown has no support for underlining, though (weird, right?). If that's the styling choice you prefer, leave it :). 


Just creating a line break will be sufficient in markdown. You don't need to manually specify line breaks with `<br />`. Did you have a formatting issue that was only resolved with `<br />`?

Reply:
We can spare the `<br>` and `<ins>` tags, it was purely meant for styling 😄.
I searched hard for a way to write an underline in md, I'm shocked there is no way 😦.

Reply:
Keep the underline! I had just incorrectly assumed it was available alongside **bolding** and _italics_! 

Did a normal line break not produce what you expected? I'm trying to figure out what the `<br />` did that wouldn't be achieved by that.

Reply:
The line break was just for adding a bit of spacing after the site is generated, it wasn't meant for the md file but for the results at the end 😄.